src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx

```diff
@@ -30,357 +30,139 @@ export function getStaticProps(context) {
-  });
-
-  return { props: { currentUser } };
-};
-```
-
-#### In `getStaticProps`
-
-Similar to static rendering with the App Router, you can pass `null` as the value of the `nextServerContext` parameter to use the Amplify Storage API with guest access.
-
-```typescript
-export async function getStaticProps() {
-  const splashUrl = await runWithAmplifyServerContext({
-    nextServerContext: null,
-    operation: (contextSpec) => getUrl(contextSpec, { key: 'splash.png' })
-  });
-
-  return {
-    props: { imageUrl: splashUrl.url.toString() },
-    revalidate: (splashUrl.expiresAt.getTime() - Date.now()) / 1000 // in seconds
-  };
-}
+})
+
+// Step 2 create the `tokenProvider`
+const tokenProvider = createUserPoolsTokenProvider(
+  amplifyConfig.Auth,
+  keyValueStorage
+);
+
+// Step 3 create the `credentialsProvider`
+const credentialsProvider = createAWSCredentialsAndIdentityIdProvider(
+  amplifyConfig.Auth,
+  keyValueStorage
+);
+
+// Execute Amplify category APIs
+// For example, calling `fetchAuthSession` on the server
+const session = await runWithAmplifyServerContext(
+  amplifyConfig,
+  {
+    Auth: { tokenProvider, credentialsProvider }
+  },
+  // The callback function that contains your business logic
+  (contextSpec) => fetchAuthSession(contextSpec),
+);
 ```
 
-## Supported APIs for Next.js server-side usage
+### Supported APIs for Server Side Usage
 
 All APIs that support use on the server are exported from the `aws-amplify/<category>/server` sub paths. You **must** use these APIs for any server side use cases.
 
-| Category | APIs | Server (Node.js) Amplify Hosting/Vercel | Vercel Edge Runtime (middleware) |
-| --- | --- | --- | --- |
-| Auth | `fetchAuthSession` | ✅ | ✅ |
-| Auth | `fetchUserAttributes` | ✅ | ✅ |
-| Auth | `getCurrentUser` | ✅ | ✅ |
-| API (GraphQL) | `generateServerClientUsingCookies` | ✅ |  |
-| API (GraphQL) | `generateServerClientUsingReqRes` | ✅ |  |
-| API (REST) | `GET` | ✅ |  |
-| API (REST) | `POST` | ✅ |  |
-| API (REST) | `PUT` | ✅ |  |
-| API (REST) | `DEL` | ✅ |  |
-| API (REST) | `HEAD` | ✅ |  |
-| API (REST) | `PATCH` | ✅ |  |
-| Storage | `getUrl` | ✅ |  |
-| Storage | `getProperties` | ✅ |  |
-| Storage | `list` | ✅ |  |
-| Storage | `remove` | ✅ |  |
-| Storage | `copy` | ✅ |  |
-
-## Migrate from Amplify JavaScript v5
-
-The Amplify JS v5 `withSSRContext` utility is no longer available with Amplify JS v6. You will need to use the `createServerRunner` function exported from `@aws-amplify/adapter-nextjs` to create a `runWithAmplifyServerContext` function, and use this function to make Amplify API calls on the server side of your Next.js app. For usage examples, see [here](#calling-amplify-category-apis-on-the-server-side).
+| Category      | APIs                  |
+| ------------- | --------------------- |
+| Auth          | `fetchAuthSession`    |
+| Auth          | `fetchUserAttributes` |
+| Auth          | `getCurrentUser`      |
+| API (GraphQL) | `generatedClient`     |
```

Comment:
nit: `generateClient` instead of generate**d**Client

Reply:
oops, good catch!